tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -847,7 +921,7 @@ def extract_arguments(start, string):
             closures["("] -= 1
         elif string[current_position] == "<":
             closures["<"] += 1
-        elif string[current_position] == ">" and string[current_position - 1] != "-":
+        elif string[current_position] == ">" and string[current_position - 1] != "-" and closures["<"] > 0:
```

Comment:
This is for handling bit shifting symbol (aka "N >> 1") in the argument.
@Jorghi12

Reply:
Awesome!